### PR TITLE
[ESLint] .tsx and lintfix

### PR DESCRIPTION
**Pull request by laemtl**
This PR:
- fixes a few warnings solvable with lintfix
- update the eslint pipeline to support .tsx files
- fixes the eslint issues found on .tsx files

### CODE REVIEW

**Comment by laemtl** [comment]
> Is the title a typo? We shouldn't have any jsx files and the JSX we have should be in js files that are already linted..

It is, thanks !

File: modules/electrophysiology_browser/jsx/electrophysiologySessionView.js [hunk 1]
```diff
@@ -320,153 +325,159 @@ class ElectrophysiologySessionView extends Component {
...
-        outputType={this.state.database[i].file.output_type}
-        />
-        </div>
-        </div>
-        </EEGLabSeriesProvider>
-        </div>
-        </FilePanel>
-        </div>
-      );
+                  electrodesURL={electrodesURL}
```
Comment (maltheism) [inline modules/electrophysiology_browser/jsx/electrophysiologySessionView.js:361]: Hi @laemtl are these props supposed to be in this PR?

File: modules/electrophysiology_browser/jsx/electrophysiologySessionView.js [hunk 2]
```diff
@@ -320,153 +325,159 @@ class ElectrophysiologySessionView extends Component {
...
+                              file.splitData.splitIndex+1
+                          )
+                        }
+                      >
+                      {'>'}
+                      </a>
+                    </>
+                  }
+                  <SeriesRenderer
+                    physioFileID={this.state.database[i].file.id}
```
Comment (maltheism) [inline modules/electrophysiology_browser/jsx/electrophysiologySessionView.js:422]: @laemtl is this change supposed to be in this PR?

Comment (driusan) [reply modules/electrophysiology_browser/jsx/electrophysiologySessionView.js:422]: #8238 is merged

**Comment by laemtl** [comment]
@driusan @maltheism I revised the scope of this PR to only address eslint tsx/lintfix issues

**Comment by maltheism** [comment]
@laemtl I'll review when the test suite passes.

**Review by maltheism — APPROVED** [review]
@laemtl Awesome work!

**Pull request merged**